webserver/preferences.js

```diff
@@ -595,7 +595,7 @@ document.addEventListener("DOMContentLoaded", () => {
 
     // Check if the current URL contains "preferences.html"
     if (currentURL.includes("preferences.html")) {
-        highlightCurrentPage(); // Highlight the current page in the navigation bar
+        //        highlightCurrentPage(); // Highlight the current page in the navigation bar
```

Comment:
Why is this commented out?

Reply:
highlightCurrentPage() function is not defined in this scope and generates a compile error.

Reply:
I assume you're referring to an error message in the browser console, not a compilation error. I haven't noticed it, but I will check after the merge (the highlightCurrentPage() function is defined in common.js, which is included with all JavaScript files).

Reply:
![IMG_20240609_204231_174.jpg](https://github.com/melkati/CO2-Gadget/assets/92800807/43d044b8-274a-420a-b16f-a2f0d7b4947a)

From development branch I have received a JavaScript error, before that function is not defined in ever .js file in development branch.